run

```diff
@@ -2,11 +2,14 @@
 
```

Comment:
I still want to see this in a bin directory with a #!/usr/bin/env ruby directive at the top...